packages/core/src/attestation/Attestation.chain.ts

```diff
@@ -17,6 +17,7 @@ import { ConfigService } from '@kiltprotocol/config'
 import { BlockchainApiConnection } from '@kiltprotocol/chain-helpers'
 import { DidUtils } from '@kiltprotocol/did'
 import Attestation from './Attestation'
+import { IDeposit } from '../common'
```

Comment:
Shouldn't this be in the `type` package?

Reply:
It's not a type, it's an interface for an on-chain data structure. All other places have these interfaces in the respective packages.